CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## v0.24.0 (Dec NN, 2019)
+
+BREAKING CHANGES:
+
+* Alpine Docker image no longer runs as root and so doesn't change ownership of the /consul-template/data and /consul-template/config directories to the consul-template user. See the [Docker Image Use](https://github.com/hashicorp/consul-template#docker-image-use) topic in the documentation for more.
```

Comment:
I visited the link for `Docker Image Use` and was brought to the top of the consul-template repo. I didn't see a docker image use topic. Not sure if I did something wrong and just wanted to flag.

Reply:
The link goes to a section that was added to the readme in this PR. It should work after the merge based on normal github markdown rules.

Reply:
ah 🤦‍♀, thank you